docs/community.md

```diff
@@ -55,8 +55,7 @@ improve existing processes and create new ones.
 
 **Learn More**
 
-* [Managing project and tool permissions and memberships](maintainers_corner.md)
-* [Legal processes](legal.md)
```

Comment:
Did you mean to remove the "Legal process" link from here?

Reply:
Yes.  The legal requirements are captured in the maintainers' page.  We may do away with legal.md or make it point to the maintainers' page.